### PR TITLE
da_b_Zant_sima OK

### PR DESCRIPTION
**Pull request by Yotona**
## CC0 License Agreement

- [X] I agree to the terms of the CC0 License.

### CODE REVIEW

File: rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp [hunk 1]
```diff
@@ -4,442 +4,238 @@
...
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-static asm void daB_ZANTS_Execute(daB_ZANTS_c* param_0) {
-    nofralloc
-#include "asm/rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima/daB_ZANTS_Execute__FP11daB_ZANTS_c.s"
+/* 80652E98-80652EB8 000538 0020+00 2/1 0/0 0/0 .text   daB_ZANTS_Execute__FP11daB_ZANTS_c */
+static void daB_ZANTS_Execute(daB_ZANTS_c* i_this) {
+    i_this->execute();
 }
```
Comment (TakaRikka) [inline rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp:130]: this should return the value as int

File: rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp [hunk 2]
```diff
@@ -4,442 +4,238 @@
...
-#pragma pop
 
-/* 80652EB8-80652EC0 000558 0008+00 1/0 0/0 0/0 .text            daB_ZANTS_IsDelete__FP11daB_ZANTS_c
- */
-static bool daB_ZANTS_IsDelete(daB_ZANTS_c* param_0) {
+/* 80652EB8-80652EC0 000558 0008+00 1/0 0/0 0/0 .text   daB_ZANTS_IsDelete__FP11daB_ZANTS_c */
+static bool daB_ZANTS_IsDelete(daB_ZANTS_c* i_this) {
     return true;
 }
```
Comment (TakaRikka) [inline rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp:135]: all main actor methods should return int

File: rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp [hunk 3]
```diff
@@ -4,442 +4,238 @@
...
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-static asm void daB_ZANTS_Delete(daB_ZANTS_c* param_0) {
-    nofralloc
-#include "asm/rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima/daB_ZANTS_Delete__FP11daB_ZANTS_c.s"
+/* 80652F18-80652F38 0005B8 0020+00 1/0 0/0 0/0 .text   daB_ZANTS_Delete__FP11daB_ZANTS_c */
+static void daB_ZANTS_Delete(daB_ZANTS_c* i_this) {
+    i_this->_delete();
 }
```
Comment (TakaRikka) [inline rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp:149]: this should return the value as int

File: rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp [hunk 4]
```diff
@@ -4,442 +4,238 @@
...
-#pragma push
-#pragma optimization_level 0
-#pragma optimizewithasm off
-static asm void daB_ZANTS_Create(daB_ZANTS_c* param_0) {
-    nofralloc
-#include "asm/rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima/daB_ZANTS_Create__FP11daB_ZANTS_c.s"
+/* 80653304-80653324 0009A4 0020+00 1/0 0/0 0/0 .text   daB_ZANTS_Create__FP11daB_ZANTS_c */
+static void daB_ZANTS_Create(daB_ZANTS_c* i_this) {
+    i_this->create();
 }
```
Comment (TakaRikka) [inline rel/d/a/b/d_a_b_zant_sima/d_a_b_zant_sima.cpp:241]: this should return the value as int

**Review by hatal175 — APPROVED** [review]

**Pull request merged**